### ISSUE TITLE
State doesn´t update proper

### ISSUE THREAD

**Issue by Blackmood667** [opravdin/weback-unofficial#20]
When checking for my battery level in home assistant the return value is 100 % until I open the app on my smartphone where it´s 46 % and the robot is stuck. After some time the state in home assistant gets the same value as the app. 

However it requires to open the app to get actual states back.

**Comment by Jezza34000** [comment]
Yes same problem for me... I didn't find any solution for this problem.

**Comment by chicco83** [comment]
I have same issue

**Comment by drronnie85** [comment]
I'm too. @opravdin you are my hero. Can you fix it?)
